### ISSUE TITLE
Seems to break swagger

### ISSUE THREAD

**Issue by iamwyza** [micro-elements/MicroElements.Swashbuckle.FluentValidation#5]
Don't exactly know why, but if I switch this on, the JSON it emits will cause things like the swagger.io editor to just hang forever.  I'm wondering if it doesn't cause some sort of referencing infinite loop.  (A requires B,  B requires C, C Requires A).

**Comment by petriashev** [comment]
Can you show me example?

**Comment by petriashev** [comment]
Works for my example:
```csharp
using FluentValidation;

namespace SampleWebApi.Contracts
{
    public class ObjectA
    {
        public string Key { get; set; }
        public ObjectB ObjectB { get; set; }
    }

    public class ObjectB
    {
        public string Key { get; set; }
        public ObjectC ObjectC { get; set; }
    }

    public class ObjectC
    {
        public string Key { get; set; }
        public ObjectA ObjectA { get; set; }
    }

    public class ObjectAValidator : AbstractValidator<ObjectA>
    {
        public ObjectAValidator()
        {
            RuleFor(sample => sample.Key).NotNull();
            RuleFor(sample => sample.ObjectB).NotNull();
        }
    }

    public class ObjectBValidator : AbstractValidator<ObjectB>
    {
        public ObjectBValidator()
        {
            RuleFor(sample => sample.Key).NotNull();
            RuleFor(sample => sample.ObjectC).NotNull();
        }
    }

    public class ObjectCValidator : AbstractValidator<ObjectC>
    {
        public ObjectCValidator()
        {
            RuleFor(sample => sample.Key).NotNull();
            RuleFor(sample => sample.ObjectA).NotNull();
        }
    }
}
```

**Comment by iamwyza** [comment]
If I try to load this into the swagger editor, it basically hangs forever.  If I disable the Fluent piece, I get a swagger json that works fine, but that doesn't have the validation.

[swagger.txt](https://github.com/micro-elements/MicroElements.Swashbuckle.FluentValidation/files/1873380/swagger.txt)

**Comment by petriashev** [comment]
Conflict in routes: /api/Account/users/{id} and /api/Account/users/{recaptchaResponse}
Change one of routes and swagger works.

**Comment by iamwyza** [comment]
Here's an updated one that passes a valid swagger.  It will load properly if you load it into the old editor as JSON, but as soon as you convert it to YAML in the new editor (which it does automatically when you load via URL), the editor breaks.

[swagger.txt](https://github.com/micro-elements/MicroElements.Swashbuckle.FluentValidation/files/1876937/swagger.txt)

**Comment by petriashev** [comment]
Last swagger works in firefox, chrome and edge.

**Comment by iamwyza** [comment]
Just tried it, nope.  Steps I'm taking:

1. https://editor.swagger.io/
2. File->Import File
3. Select the file and click upload
4. Swagger loads, but browser locks up (try scrolling or clicking on any of the endpoints).  Tested with both Chrome and Edge.

I can make a video of the steps too if you'd like, but I can reproduce it 100% of the time using this approach.

**Comment by petriashev** [comment]
I copied your text from last swagger.txt to left pane in new swagger editor and it works in browsers

**Comment by petriashev** [comment]
Do you still have a problem?

**Comment by iamwyza** [comment]
Yes.  As noted, loading the json via copy works.  But the editor will auto-convert to YAML when it loads via url or via file Import.  When the editor converts to YAML it breaks.

**Comment by petriashev** [comment]
Looks like js hangs on yaml parsing in editor.swagger.com
I think you need to add issue with your swagger.txt to https://github.com/swagger-api/swagger-editor/issues
Is your swagger work in net core swagger ui?

**Comment by petriashev** [comment]
See comment: https://github.com/swagger-api/swagger-editor/issues/1734#issuecomment-385068928
